api/db/seeds/data/campaigns-pro-builder.js

```diff
@@ -100,6 +100,23 @@ __Plus d'infos :)__
     multipleSendings: true,
   });
 
+  databaseBuilder.factory.buildCampaign({
+    id: 27,
+    name: 'Pro - Campagne Pix CNAV',
+    code: 'PIXCNAV01',
+    type: 'ASSESSMENT',
+    organizationId: PRO_CNAV_ID,
+    creatorId: 2,
+    ownerId: 2,
+    targetProfileId: TARGET_PROFILE_PIX_EMPLOI_CLEA_ID,
```

Comment:
Changer pour un targetProfile non POLE EMPLOI 
```suggestion
    targetProfileId: TARGET_PROFILE_PIX_CNAV_CLEA_ID,
```